python/runtime/pai/submitter_test.py

```diff
@@ -33,8 +33,8 @@ def test_get_oss_model_url(self):
 
     def test_get_pai_tf_cmd(self):
         conf = get_cluster_config({})
-        os.environ[
-            "SQLFLOW_OSS_CHECKPOINT_CONFIG"] = '{"arn":"arn", "host":"host"}'
+        os.environ["SQLFLOW_OSS_CHECKPOINT_CONFIG"] = \
```

Comment:
Why use this format instead of standard JSON format `'''{"arn":"arn", "host":"host"}'''`?

Reply:
Couler generates the env like `'''{json object}'''`.